csource/common.go

```diff
@@ -203,6 +225,8 @@ static void execute_command(const char* format, ...)
 
 int tunfd = -1;
 
+#define TUN_MTU 1500
```

Comment:
Ah, I see. The packet will be truncated. I've changed the buffer size to 64 << 10.